pom.xml

```diff
@@ -95,6 +95,7 @@
         <jetty.version>9.2.10.v20150310</jetty.version>
         <pax.web.version>4.2.4</pax.web.version>
         <pax.cdi.version>0.12.0</pax.cdi.version>
+        <pax.logging.version>1.8.5</pax.logging.version>
```

Comment:
Why did we need to do this?  Looks like Karaf 4.0.4 is already including this version.

https://issues.apache.org/jira/browse/KARAF-4239